File: Day22/occurence.cpp
#include<iostream>
using namespace std;
int main(){
    int arr[5];
    for(int i=0;i<5;i++){
        cin>>arr[i];
    }
    cout<<endl;
    cout<<"The array entered is :"<<endl;
    for(int i=0;i<5;i++){
        cout<<arr[i]<<" ";
    }
    int n;
    int count=0;
    cout<<"Enter the element which you want to search for :";
    cin>>n;
    for(int i=0;i<5;i++){
        if(arr[i]==n){
            count++;
        }
    }
    cout<<"The occurence of "<<n<< "is "<<count<<" times";
}

File: Day66/Occurence.java
import java.util.Scanner;

public class Occurence {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the size of the array :");
        int size=sc.nextInt();
        int count=0;
        System.out.println("Enter the array of size :"+size);
        int[] arr=new int[size];
        for(int i=0;i<size;i++){
            arr[i]=sc.nextInt();
        }
        System.out.println("Enter the number you want to find occurence of:");
        int num=sc.nextInt(); //to find occurence of 
        for(int i=0;i<size;i++){
            if(arr[i]==num){
                count++;
            }
        }
        System.out.println("The occurence of "+num+" is "+count);
        sc.close();

     }
}
